CCSocialShare/android/java/src/org/kawaz/socialshare/SocialShare.java

```diff
@@ -15,8 +16,23 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.util.List;
+import java.util.Calendar;
```

Comment:
Is this needed?

Reply:
TwitterKit cannot share the same filename.  I use Calendar for rename by the time.